e2e/interchaintestv8/ibc_eureka_test.go

```diff
@@ -1191,76 +1237,24 @@ func (s *IbcEurekaTestSuite) createICS20MsgSendPacket(
 	}
 }
 
-func (s *IbcEurekaTestSuite) getCommitmentProof(ctx context.Context, path []byte) []byte {
-	eth, simd := s.EthChain, s.CosmosChains[0]
-
-	storageKey := ethereum.GetCommitmentsStorageKey(path)
-	storageKeys := []string{storageKey.Hex()}
-
-	blockNumberHex := fmt.Sprintf("0x%x", s.LastEtheruemLightClientUpdate)
-	proofResp, err := eth.EthAPI.GetProof(s.contractAddresses.IbcStore, storageKeys, blockNumberHex)
+func (s *IbcEurekaTestSuite) BroadcastSdkTx(ctx context.Context, chain *cosmos.CosmosChain, user ibc.Wallet, gas uint64, txBodyBz []byte) *sdk.TxResponse {
+	var txBody txtypes.TxBody
+	err := proto.Unmarshal(txBodyBz, &txBody)
 	s.Require().NoError(err)
-	storageProof := proofResp.StorageProof[0]
 
-	if s.rustFixtureGenerator.ShouldGenerateFixture() {
-		_, ethereumClientState := s.GetEthereumClientState(ctx, simd, s.EthereumLightClientID)
-		_, ethereumConsensusState := s.GetEthereumConsensusState(ctx, simd, s.EthereumLightClientID, clienttypes.Height{
-			RevisionNumber: 0,
-			RevisionHeight: s.LastEtheruemLightClientUpdate,
-		})
-
-		s.rustFixtureGenerator.AddFixtureStep("commitment_proof", &ethereumtypes.CommitmentProof{
-			Path:           fmt.Sprintf("0x%x", path),
-			StorageProof:   storageProof,
-			ProofSlot:      s.LastEtheruemLightClientUpdate,
-			ClientState:    ethereumClientState,
-			ConsensusState: ethereumConsensusState,
-		})
-	}
-
-	bz, err := json.Marshal(&storageProof)
-	s.Require().NoError(err)
-
-	return bz
-}
-
-func (s *IbcEurekaTestSuite) updateClientAndMembershipProof(
-	ctx context.Context,
-	counterpartyChain *cosmos.CosmosChain,
-	proofType operator.SupportedProofType,
-	ibcProofPaths [][]byte,
-) (*ics26router.IICS02ClientMsgsHeight, []byte) {
-	clientState, err := s.sp1Ics07Contract.GetClientState(nil)
-	s.Require().NoError(err)
-
-	trustedHeight := clientState.LatestHeight.RevisionHeight
-	latestHeight, err := counterpartyChain.Height(ctx)
-	s.Require().NoError(err)
+	var msgs []sdk.Msg
+	for _, msg := range txBody.Messages {
+		var sdkMsg sdk.Msg
+		err = chain.Config().EncodingConfig.InterfaceRegistry.UnpackAny(msg, &sdkMsg)
+		s.Require().NoError(err)
 
-	proofPaths := make([][][]byte, len(ibcProofPaths))
-	for i, path := range ibcProofPaths {
-		proofPaths[i] = [][]byte{
-			[]byte("ibc"),
-			path,
-		}
+		msgs = append(msgs, sdkMsg)
 	}
 
-	proofPathsStr := operator.ToBase64KeyPaths(proofPaths...)
+	s.Require().NotZero(len(msgs))
 
-	args := append([]string{
-		"--trust-level", testvalues.DefaultTrustLevel.String(),
-		"--trusting-period", strconv.Itoa(testvalues.DefaultTrustPeriod),
-		"--base64",
-	},
-		proofType.ToOperatorArgs()...,
-	)
-	proofHeight, ucAndMemProof, err := operator.UpdateClientAndMembershipProof(
-		uint64(trustedHeight), uint64(latestHeight), proofPathsStr, args...,
-	)
+	resp, err := s.BroadcastMessages(ctx, chain, user, gas, msgs...)
```

Comment:
It looks like the PoS tests for 50 transfers fail here due to lack of gas: https://github.com/cosmos/solidity-ibc-eureka/actions/runs/12752998454/job/35546992694